### PR TITLE
Feature/ui options

### PR DESCRIPTION
**Pull request by SebastianStehle**
Introduce settings that are added to Index.html
Option to hide news dialog.
Option to hide onboarding tooltips.
Option to redirect to login page automatically without extra click.

### CODE REVIEW

File: src/Squidex/Areas/Api/Controllers/UI/Models/UISettingsDto.cs [hunk 1]
```diff
@@ -3,26 +3,12 @@
 // ==========================================================================
 //  Copyright (c) Squidex UG (haftungsbeschränkt)
 //  All rights reserved. Licensed under the MIT license.
-// ==========================================================================
-
-using System.ComponentModel.DataAnnotations;
+// ==========================================================================using System.ComponentModel.DataAnnotations;
```
Comment (Jason-Segal) [inline src/Squidex/Areas/Api/Controllers/UI/Models/UISettingsDto.cs]: Should this line be removed since it is commented out?

Comment (SebastianStehle) [reply src/Squidex/Areas/Api/Controllers/UI/Models/UISettingsDto.cs]: Fixed

File: src/Squidex/app/shared/guards/must-be-authenticated.guard.spec.ts [hunk 2]
```diff
@@ -9,24 +9,25 @@ import { Router } from '@angular/router';
...
     let authService: IMock<AuthService>;
-    let authGuard: MustBeAuthenticatedGuard;
+    let uiOptions = new UIOptions({ map: { type: 'OSM' } });
+    let uiOptionsRedirect = new UIOptions({ map: { type: 'OSM' }, redirectToLogin: true });
 
     beforeEach(() => {
         router = Mock.ofType<Router>();
 
         authService = Mock.ofType<AuthService>();
-        authGuard = new MustBeAuthenticatedGuard(authService.object, router.object);
```
Comment (Jason-Segal) [inline src/Squidex/app/shared/guards/must-be-authenticated.guard.spec.ts]: Just out of interest, how comes this was moved out of the 'BeforeEach' section if it is used in all tests within this file?

Comment (SebastianStehle) [reply src/Squidex/app/shared/guards/must-be-authenticated.guard.spec.ts]: It is used with different arguments.

**Pull request merged**